cluster/ci/monitoring/mixins/grafana_dashboards/ghproxy.jsonnet

```diff
@@ -159,4 +192,76 @@ dashboard.new(
     x: 20,
     y: 0,
   })
+.addPanel(
+    (graphPanel.new(
+        'Token Usage',
+        description='GitHub token usage by token identifier and API version.',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_current=true,
+        min='0',
+        max='5000',
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'label_replace(sum(github_token_usage) by (api_version, token_hash), "token_hash_short", "$1", "token_hash", "([a-z0-9]{5})(.*)")',
+         legendFormat='{{api_version}}:{{token_hash_short}}',
+    )), gridPos={
+    h: 9,
+    w: 24,
+    x: 0,
+    y: 18,
+  })
+.addPanel(
+    (graphPanel.new(
+        'Request Rates',
+        description='GitHub request rates by token identifier, path and status.',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_current=true,
+        legend_sort='current',
+        legend_sortDesc=true,
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'label_replace(sum(rate(github_requests[5m])) by (token_hash, path, status), "token_hash_short", "$1", "token_hash", "([a-z0-9]{5})(.*)")',
+         legendFormat='{{api_version}}:{{token_hash_short}}:{{status}}',
+    )), gridPos={
+    h: 9,
+    w: 24,
+    x: 0,
+    y: 18,
+  })
+.addPanel(
+    (graphPanel.new(
+        'latency percentile with token_hash ${token_hash}, path ${path}, and status ${status}',
```

Comment:
Let's try to make these a little more user friendly and less implementation dependent:

```
Latency Distribution for ${status} ${path}
```

Reply:
Done.